test/e2e/aws_test.go

```diff
@@ -406,7 +517,7 @@ func makeAWSMachine(namespace, name string) {
 			Namespace: namespace,
 		},
 		Spec: infrav1.AWSMachineSpec{
-			InstanceType:       "t3.large",
+			InstanceType:       "m5.large",
```

Comment:
This should probably be exposed as a parameter for `makeAWSMachine` to limit costs for running the tests contained in `aws_test.go` compared to `conformance_test.go`

Reply:
Sounds good. Will do.